api/envoy/extensions/filters/http/grpc_json_transcoder/v3/transcoder.proto

```diff
@@ -222,6 +222,10 @@ message GrpcJsonTranscoder {
   // This is to support `HTML 2.0 <https://tools.ietf.org/html/rfc1866#section-8.2.1>`_
   bool query_param_unescape_plus = 12;
 
+  // If true, try to match the custom verb even if it is unregistered. By
+  // default, only match when it is registered.
+  bool match_unregistered_custom_verb = 13;
```

Comment:
may need to add more comment to explain this field
* add a link to the http.rule syntax
* a custom verb is the last ":" without slash in between
* if a template with a custom_verb,   a request should have extra custom_verb to match it.
* use a example to explain it
* a request   "/foo/bar1:bar2",  it used to match /foo/{x=*},  and x=bar1:bar2.  
* with this flag on, above will not match any more.